plugin/files/airconsole/runtime.js

```diff
@@ -77,7 +77,9 @@ cr.plugins_.AirConsole = function(runtime)
           self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessageKey, self);
         }
         if (data.message) {
-          self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessage, self);
+          self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessageIs, self);
+		  self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessageFrom, self);
+		  self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessage, self);
```

Comment:
Well, I want to know if we receive a message, from whoever it is and whatever it contains

http://puu.sh/rM0gJ.png

Original OnMessage restrics  in a way that the message reception only triggers if we know both other conditions, DeviceId and Message. With these two, we can trigger  actions if a specific device sent a message, or if anybody sent anything. Applications? Well, say I build a pictionary game, first to answer wins, my controllers will send strings back to the screen and onMessage will trigger. I can't do that with the old implementation, I need to know the message and the deviceId